server/.env.example

```diff
@@ -1,5 +1,6 @@
 # Use this for local setup
 PG_DATABASE_URL=postgres://twenty:twenty@localhost:5432/default?connection_limit=1
+PRISMA_DATABASE_URL=postgres://twenty:twenty@localhost:5432/default?connection_limit=1
```

Comment:
Why are we creating 2 env variables? It seems they are both exposed the same way?

Reply:
One is for the source database using prisma, the other one is for the target database using typeorm